sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/FileFormat.scala

```diff
@@ -264,9 +261,11 @@ object FileFormat {
       fileSize: Long,
       fileBlockStart: Long,
       fileBlockLength: Long,
-      fileModificationTime: Long): InternalRow = {
+      fileModificationTime: Long,
+      otherConstantMetadataColumnValues: Map[String, Any]): InternalRow = {
```

Comment:
How is `otherConstantMetadataColumnValues` generated? `FileFormat` doesn't have a API for it.